### DISCUSSION TITLE
Deleting notes

### DISCUSSION THREAD

**Discussion by schwa** [ohare93/brain-brew#46] [Q&A]
How does brain-brew handle deleting of notes?

Rows removed from my csv don't seem to be matched by removed of cards/notes in genki…

I'm resorting to adding a "DELETE_ME" tag on notes I want to delete.

**Comment by jeprecated** [comment]
Deleted notes are not handled, at the moment. Though it would be great for Brain Brew to handle it.

The issue is that CrowdAnki has no functionality for deletion, and Brain Brew is reliant on CrowdAnki for interactions with Anki itself. It is a very very tricky subject to handle, as a record of which notes should be deleted needs to be kept somewhere. Either that or CrowdAnki should offer to delete all cards in an import that are not in the deck.

Tagging your items for deletion is a great workaround :+1:
